box-ui/src/components/css/FocusMode.css

```diff
@@ -10,23 +10,36 @@
     z-index: 2;
     pointer-events: none;
     transition: all 0.5s linear;
+    background-clip: content-box;
+    background-color: rgba(199, 41, 41, 0.473);
+    animation: blinkingBackground 2s infinite;
+}
+
+@keyframes blinkingBackground {
+    0% {
+        background-color: rgba(255, 255, 0, 0.5);
+    }
+    50% {
+        background-color: rgba(255, 255, 0, 0);
+    }
+    100% {
+        background-color: rgba(255, 255, 0, 0.5);
+    }
 }
 
 #explanationPosition {
+    z-index: 3;
```

Comment:
Judging by the other numbers that were used for the z-index values, you might want to increase it to 300 or else it might be unssen...